### ISSUE TITLE
Provide event driven subscriptions

### ISSUE THREAD

**Issue by avanc** [planeworld/pwng-server#3]
Currently, the sim state (running/stopped) is included in the simulatin statistics. As the statistics is not that crucial for me, I would subscribe it with 0.1Hz.
However, the sim state is used for e.g. changing the UI (hide button for starting or stopping sim) and it might become annoying if the UI is wrong for almost 10 seconds.
Thus, it would be great if seldomly changing information is published differently only on change.

**Comment by bfeldpw** [comment]
Yes, event-based subscriptions should be implemented, and yes, server-state should be separated from UI, but apart from that: what stops you from subscribing the 10Hz or 5Hz topic instead for now?

**Comment by avanc** [comment]
Currently, it is done as you proposed.
